Add RDBMS article

RyanBuck182 opened the pull request:
## Proposed changes

Add an RDBMS article.

## Types of changes

What types of changes does your content introduce to this project?
_Put an `x` in the boxes that apply (without "`")_


- [ ] Proofreading (fix spelling, grammar, punctuation, accuracy) [Proofreading](https://www.scribbr.com/language-rules/what-is-proofreading/)
- [X] New content (new section, new readme, etc.)
- [ ] Modifications (modifying existing content)
- [ ] Other (please describe)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. Don't hesitate to ask if you're unsure about any of them. We're here to help! This simply reminds us of what we will look for before merging your code._
_Note: If you submit code make sure to test it_


- [ ] Tested any code that was added/modified in this PR
- [ ] Added external links
- [X] Added internal links
- [X] Added images
- [ ] Added videos
- [ ] Added code snippets


## Further comments

Let me know if there are any questions.

## Adding your name to the contributor's list
Comment "@all-contributors please add @your_username for content" to add your name to the contributor's list.
_You can change the "content" with any of [this](https://allcontributors.org/docs/en/emoji-key)_

### Reviewers: 
- [ ] @shhossain 


_Please ensure that you are not adding any sensitive information like API keys, secrets, passwords, tokens, usernames, or other credentials. Refrain from adding your name, email, or other personal information._

RyanBuck182 commented:
https://github.com/all-contributors please add @RyanBuck182 for content

shhossain reviewed (changes requested):
You need a `readme.md` in DBMS folder which links to `Relational Model/readme.md`

RyanBuck182 commented:
> You need a `readme.md` in DBMS folder which links to `Relational Model/readme.md`

I moved the general DBMS readme file into the main DBMS folder because the subfolder for it was unnecessary. Then I added links in it to the READMEs for the various database models.

shhossain reviewed (approved):
Nice work.
